### PR TITLE
docs(ledger-plugin): fix some minor errors on documentation

### PR DESCRIPTION
**Pull request by takeutak**
Signed-off-by: Takuma TAKEUCHI <takeuchi.takuma@fujitsu.com>

- related with Issue #742

### CODE REVIEW

**Review by sfuji822 — APPROVED** [review]
LGTM

File: docs/manual-of-auto-testing-codes.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# The manual of auto-testing codes
```
Comment (petermetz) [inline docs/manual-of-auto-testing-codes.md:1]: @takeutak Any chance that you could use the default test script and then there's no need for this document?

**Review by petermetz — CHANGES_REQUESTED** [review]

File: docs/manual-of-auto-testing-codes.md (hunk 1 above)
Comment (takeutak) [reply docs/manual-of-auto-testing-codes.md:1]: The doc was already put on this file <https://github.com/hyperledger/cactus/blob/main/packages/manual-of-auto-testing-codes.md>. I intended to move this file to docs directory, but I missed to delete the one on the original directory.
On another PR, I will delete the one on the original directory to fix it.

Comment (petermetz) [reply docs/manual-of-auto-testing-codes.md:1]: @takeutak Thanks for letting me know. Do you have any plans to migrate to the default test scripts that get invoked by Lerna for the combined coverage reports?

Comment (takeutak) [reply docs/manual-of-auto-testing-codes.md:1]: I do not have any plan yet. However, I think combining it with existing CI checking tools to make it easier to use is one of the ways to improve it.
I think it is appropriate to discuss this topic in another issue. Could you approve this PR for now?

Comment (petermetz) [reply docs/manual-of-auto-testing-codes.md:1]: @takeutak Awesome, thanks.

**Review by petermetz — APPROVED** [review]

**Pull request merged**